Pop-up box

Lu-Dai opened the pull request.

Lu-Dai committed:
pop-up box

Lu-Dai committed:
pop-up box 2

Lu-Dai committed:
remove .idea

jsclose committed:
Javascript object, UI, Start Menu, displayquestion

ZixinLiu committed:
sample.css

ZixinLiu committed:
Sample commit

Showing how to commit

ZixinLiu committed:
Recommit Sample.css

Lu-Dai committed:
Add Text

sshusain reviewed (changes requested):
JS imports should be in bottom of body, not in head. see other examples

JS code should be in a .js file, not embedded inside the html

jsclose committed:
Appended gitignore

jsclose committed:
Feat:front end

Created question object.
Created display Question function
Created get random question function given an id
created main driver function

jsclose committed:
Feat: front end

Implemented start menu pop up
Created Category picker start up
Worked with Mark to create question, score, category, answers part.

jsclose committed:
Updated quiz app directory

sshusain committed:
Merge branch 'master' into master

jsclose committed:
Merge pull request #2 from ZixinLiu/master

Merging Css files into directory

jsclose committed:
Merge remote-tracking branch 'SOCR/master' into pr/1

jsclose committed:
Merge pull request #1 from Lu-Dai/master

Reconciliation

jsclose committed:
Merge pull request #3 from jsclose/pr/1

Pr/1

Lu-Dai committed:
Merge pull request #1 from jsclose/master
